janusgraph-inmemory/src/test/java/org/janusgraph/graphdb/inmemory/InMemoryGraphTest.java

```diff
@@ -57,6 +57,9 @@ public void clopen(Object... settings) {
     @Override @Test @Disabled
     public void testLocalGraphConfiguration() {}
 
+    @Override @Test @Disabled
+    public void testLimitBatchSizeForMultiQuery() {}
```

Comment:
Can I disable Codacy somehow? Alternative would be to add an `assertTrue(true)` to every disabled test.

Reply:
I disabled this check in Codacy. It shouldn't appear again I think. That said, when Codacy check fails, it doesn't block PR from being merged. Codacy makes suggestions which are optional to be resolved. It's perfectly fine when someone doesn't agree with Codacy's report, in such cases we just ignore Codacy. That said, most of the time it finds some places for cleanup (like unused imports and so on).